src/components/AAPModal/__tests__/AnsibleAutomationPlatformModal.spec.tsx

```diff
@@ -127,10 +152,16 @@ describe('AnsibleAutomationPlatformModal', () => {
     });
     getAAPDataMock.mockReturnValue(mockedAAPCR);
     act(() => {
-      render(<AnsibleAutomationPlatformModal onClose={mockCallBack} initialStatus="unknown" />);
+      render(
+        <AnsibleAutomationPlatformModal
+          onClose={mockCallBack}
+          initialStatus="unknown"
+          provisioningLabel={'provisioning'}
```

Comment:
@mfrancisc we duplicate this label in both the test and source code. Should we make a constant for the labels and fallback labels?

Reply:
good point, I've cleaned them up in https://github.com/RedHatInsights/developer-sandbox-ui/pull/67/commits/836ef45b9442fd2c459a735a5b09f710e7b1c76e